docs/source/running-autometa.rst

```diff
@@ -2,8 +2,28 @@
 Running Autometa
 ================
 
+
 Before running anything make sure you have activated the conda environment using
- ``conda activate autometa`` and are in the Autometa directory.
+
+``conda activate autometa`` and are in the Autometa directory.
+
+See the install page for details on setting up your conda environment.
+
+
+Testing Autometa
+================
+
+.. code-block:: shell
```

Comment:
Won't adding `bash` make it more specific

Reply:
shell or bash, this highlight will not make any difference